data/notebooks/ingredients.py

```diff
@@ -159,9 +183,9 @@ def reverse(d):
     "database": "Base de données",
     "search": "Termes de recherche",
     "default_origin": "Origine par défaut",
-    "category": "Catégorie de procédé",
+    "process_categories": "Catégories de procédé",
     # ingredients attributes
-    "categories": "Catégories d'ingrédient",
+    "ingredient_categories": "Catégories d'ingrédient",
```

Comment:
```suggestion
    "ingredient_categories": "Catégories d'ingrédients",
```
?

Reply:
Same here, you're editing a single ingredient